src/components/PersonCard.astro

```diff
@@ -10,7 +10,8 @@ export interface Props {
 
 const { personMarkdown, variant = "large" } = Astro.props as Props;
 
-const { frontmatter: person, file: id } = personMarkdown;
+const { frontmatter: person, file } = personMarkdown;
+const id = file.split("/").pop() || "";
```

Comment:
the id was unnecessary long (project/src/data/....) , the file name should be enough. in case we have duplicate names in different folders, we can add another level in front, but not the full file name